.goreleaser.yaml

```diff
@@ -96,8 +96,12 @@ dockers:
       - --platform=linux/amd64
     image_templates:
       - "flashbots/suave-geth:{{ .ShortCommit }}-amd64"
-      - "flashbots/suave-geth:{{ .Tag }}-amd64"
-      - "flashbots/suave-geth:latest-amd64"
+      - "flashbots/suave-geth:v{{ .Version }}-amd64"
+      {{- if not .IsNightly }}
```

Comment:
can you remove this inversion this for less cognitive load? i.e. instead of `{{- if not .IsNightly }}` do `{{- if .IsNightly }}`

Reply:
😢 but I so love double negations 😢